CONTRIBUTING.md

```diff
@@ -55,8 +55,14 @@ The easiest way to get set up for development is to run the following script:
 # then clone your fork
 git clone https://github.com/<your-github-username>/gluon-ts.git
 
-# enter the cloned repo and run
+# enter the cloned repo
+cd gluon-ts
+
+# run development setup
 ./dev_setup.sh
+
+# install GluonTS from current source with shell dependencies
+pip install -e ".[shell]"
```

Comment:
I think `shell` isn't strictly required for a basic setup, but I still think it's good to install as well.